pym/_emerge/BinpkgExtractorAsync.py

```diff
@@ -38,6 +39,15 @@ def _start(self):
 			self.returncode = 1
 			self._async_wait()
 			return
+		if subprocess.call("type" + decomp_cmd, shell=True,
```

Comment:
The calling code is asynchronous, so this subprocess call needs to be asynchronous as well. In fact, BinpkgExtractorAsync inheris from SpawnProcess specifically for this purpose. You should assign self.args and then call SpawnProcess._start(self) like this code in this file that immediately follows.